src/screens/points/content/PointsContent.tsx

```diff
@@ -735,7 +735,7 @@ export function PointsContent() {
             <Box gap={24}>
               {rewardsEnabled && (showClaimYourPoints || showMyEarnings) && (
                 <Box gap={20}>
-                  {showClaimYourPoints && !isReadOnlyWallet && <ClaimCard claim={claimableBalance.display} value={claimablePrice} />}
```

Comment:
i decided to enable the ClaimCard (eth rewards) for read only wallets so i can also display eth rewards in the claimables section of the wallet screen for read only wallets. since that is the default behavior for claimables in general